### DISCUSSION TITLE
webBase/CSS/%E5%88%9D%E5%A7%8B%E5%8C%96%E6%A0%B7%E5%BC%8F%E7%AE%80%E5%8D%95%E7%89%88

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#338] [Announcements]
# webBase/CSS/%E5%88%9D%E5%A7%8B%E5%8C%96%E6%A0%B7%E5%BC%8F%E7%AE%80%E5%8D%95%E7%89%88

初始化css样式简单版

http://banmagnas.fun:9822/webBase/CSS/%E5%88%9D%E5%A7%8B%E5%8C%96%E6%A0%B7%E5%BC%8F%E7%AE%80%E5%8D%95%E7%89%88.html

**Comment by mengyuan1997** [comment]
加油

**Reply by mengyuan1997** [reply]
1
